src/components/SnazzyButton.tsx

```diff
@@ -3,36 +3,28 @@ import { createUseStyles } from 'react-jss';
 import { snazzyButtonStyle } from './helpers';
 
 
-interface ButtonProps {
-  children: React.ReactNode;
-}
+
 
 const styles = {
-  snazzyButton: {
+  snazzyButton: snazzyButtonStyle
+
 
-  }
 };
 
 const useStyles = createUseStyles(styles);
 
 interface SnazzyButtonProps {
   isSnazzy: boolean;
+  children: React.ReactNode;
 }
 
 const SnazzyButton: React.FC<SnazzyButtonProps> = ({ isSnazzy, children, ...rest }) => {
   const classes = useStyles();
 
-  const classNames = {
-    [classes.snazzyButton]: isSnazzy,
-
-  };
 
-  const className = Object.keys(classNames)
-    .filter((key) => classNames[key])
-    .join(' ');
 
   return (
-    <button className={className} {...rest}>
+    <button className={isSnazzy ? classes.snazzyButton : ' '} {...rest}>
```

Comment:
You are using the default "button" component for react. 
From the task: 
> _this new component will use the Patternfly Button component as a base_

Import and use the "Button" component from Patternfly here.

Reply:
Import and **_### use_** the "Button" component from Patternfly here.